src/deploy-commands.ts

```diff
@@ -0,0 +1,73 @@
+import { REST, Routes, Client } from 'discord.js';
```

Comment:
Here is the Client discord's type, you can use it in here: https://github.com/TeamNovaSoft/novabot/pull/112/files#r1949021235